content/docs/command-reference/remote/modify.md

```diff
@@ -354,6 +361,13 @@ authentication method:
   $ dvc remote modify --local myremote connection_string 'mysecret'
   ```
 
+- `account_name` - storage account name. Required if `coonection_string` is not
+  supplied. Will accompany the options below:
```

Comment:
"Will accompany the options" - not sure what this means. Is it an instruction ("must accompany")? An option ("may accompany")? A request ("should accompany")? Something else?

Reply:
This relates to my Q in https://github.com/iterative/dvc.org/pull/2489#pullrequestreview-665102806. I think it means `account_name` is almost always required now but let's see what @isidentical says.

Reply:
BTW we seem to be missing a bullet entry for `connection_string` ???

Reply:
> I think it means account_name is almost always required now but let's see what @isidentical says.

Yes, that is the case.

Reply:
> BTW we seem to be missing a bullet entry for connection_string ?

This bullet partially covers it, but if you'd like to add it as a separate one, sure!

Reply:
>  you'd like to add it as a separate one, sure

Please 🙂

Reply:
> almost always required

not at all clear. If it is merely "almost" then we shouldn't open with a statement which strongly implies "absolutely always" and then go on to contradict ourselves later by saying that it's optional.

I'd recommend leaving this out

```suggestion
  supplied.
```

"Will accompany the options below" sounds a bit like "works in tandem with the options below" which in turn sounds like "CLI options affect each other" which in turn sounds like either redundant info or unneeded implementation detail.

Reply:
Up to you here @casperdcl, as long as things are clear. I agree to remove this sentence but maybe there's some way to indicate in each param which other params it works with?

Reply:
maybe a table?

Reply:
Sure we have a few e.g. https://dvc.org/doc/user-guide/project-structure/pipelines-files#stage-entries